### ISSUE TITLE
Fix "Similar code" issue in js/test5a.js

### ISSUE THREAD

**Issue by ErnieAtLYD** [Code-for-Miami/When-Will-It-Flood-#6]
Similar code found in 9 other locations (mass = 290)

https://codeclimate.com/github/Code-for-Miami/When-Will-It-Flood-/js/test5a.js#issue_5973dfc001111a000100021a

**Comment by susanfiu** [comment]
Hey Chris and Ernie - I would add Pablo but I can't find his e-mail address!


Thanks so much for this. You know I am a GitHub idiot. Just one thing to note - it looks like some of the identical code is coming from the dozens of iterations of the project that I published in miamistories.net/susan. Ernie, I am happy to work on the ReadMe. I did download Pablo's changes to make the dropdown work, and edited it in Dreamweaver to change the styles and make a few other adjustments. I will hold off on trying to - reconcile? - it with the existing GitHub files until I see you all on Monday because I am a GitHub idiot. Some of the code is needlessly repetitive (see LoadUpTheMonths) because I don't know how to create an array within an array in Javascript.


sj


Susan Jacobson, PhD
School of Communication + Journalism
Florida International University
susan.jacobson@fiu.edu
________________________________
